releasenotes/notes/54644.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 54644
+releaseNotes:
+- |
+  **Fixed** excessive iptables info-level log entries for rule checks and deletions.
```

Comment:
nit: good to mention that you can get these back/see all iptables mutations if you enable debug logging.